lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -106,7 +106,8 @@ export enum ApiId {
 export enum InteractionType {
     Redirect = "redirect",
     Popup = "popup",
-    Silent = "silent"
+    Silent = "silent",
+    None = "none"
```

Comment:
This change also looks unrelated to the intent of your PR.

Reply:
also related to enums available for downstream packages